### PR TITLE
Update repo for Fondant 0.8.dev6

### PR DESCRIPTION
**Pull request by RobbeSneyders**

### CODE REVIEW

File: src/pipeline_eval.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 """Fondant pipeline to evaluate a RAG pipeline."""
+import pyarrow as pa
 from fondant.pipeline import Pipeline
```
Comment (Hakimovich99) [inline src/pipeline_eval.py:5]: Should add a `api_keys: dict = {}` argument in the create_pipeline function. Same in the pipeline_index

File: src/pipeline_eval.py [hunk 2]
```diff
@@ -45,7 +47,7 @@ def create_pipeline(  # noqa: PLR0913
     )
```
Comment (Hakimovich99) [inline src/pipeline_eval.py:47]: Add `"api_keys": api_keys` in arguments. Same in pipeline_index.

**Review by Hakimovich99 — COMMENTED** [review]
Thanks @RobbeSneyders! A quick comment for the pipeline script.

File: src/pipeline_eval.py (hunk 2 above)
Comment (RobbeSneyders) [reply src/pipeline_eval.py:47]: Let's add this as a separate change.

**Review by Hakimovich99 — APPROVED** [review]

**Pull request merged**